### ISSUE TITLE
Better appdata summary

### ISSUE THREAD

**Issue by bertob** [flxzt/rnote#99]
The current summary "A simple drawing application to create handwritten notes" isn't great (longer than 35 characters, not a call to action, unnecessary generic adjectives).

I'd suggest something shorter and simpler, like "Take handwritten notes".

More general guidance here: https://gitlab.gnome.org/GNOME/Initiatives/-/wikis/App-Metadata

**Comment by flxzt** [comment]
I already have a history of changing that slogan. I had "take handwritten notes", until I discovered that xournal++ has almost the same, then I had "create handwritten notes with ease", until I looked at Xournal++ website, which has the combination of both. I settled on this longer one because I don't want rnote to look like a copy-cat (which in hindsight I guess it is, but I came to a lot of decisions and conclusions the same as xournal++, without looking at their app). So I would prefer a better slogan, but I also want it to be a little more original, if you have other suggestions?
